templates/navigation/_dropdown.html

```diff
@@ -29,76 +29,58 @@
-                  <li class="p-navigation--list-heading">
-                    <h2 class="p-muted-heading" data-level="2">{{ links_section.title }}</h2>
-                  </li>
-                {% endif %}
-                {% for link in links_section.links %}
-                  <li>
-                    <a href="{{ link.url }}"
-                       class="p-navigation__dropdown-item"
-                       data-level="2">
-                      <strong>{{ link.title }}</strong>
-                      <br />
-                      <small class="u-text--muted">{{ link.description }}</small>
-                    </a>
-                  </li>
+              {% if nav_section.primary_links %}
+                {% for link in nav_section.primary_links.links %}
+                  {% with link = link, data_level = "2" %}
+                    {% include "navigation/partials/_list-item.html" %}
+                  {% endwith %}
                 {% endfor %}
-              {% endfor %}
+              {% endif %}
               {% if nav_section.secondary_links %}
-                {% for links_section in nav_section.secondary_links %}
-                  <li class="p-navigation--list-heading">
-                    <h2 class="p-muted-heading is-muted" data-level="2">{{ links_section.title }}</h2>
+                {% if nav_section.secondary_links.title %}
+                <li class="p-navigation--list-heading">
+                  <h2 class="p-muted-heading is-muted" data-level="2">{{ nav_section.secondary_links.title }}</h2>
+                </li>
+                {% endif %}
+                {% for link in nav_section.secondary_links.links %}
+                  <li>
+                    <a class="p-navigation--list-link" href="{{ link.url }}" data-level="2">{{ link.title }}</a>
                   </li>
-                  {% for link in links_section.links %}
-                    <li>
-                      <a class="p-navigation--list-link" href="{{ link.url }}" data-level="2">{{ link.title }}</a>
-                    </li>
-                  {% endfor %}
                 {% endfor %}
               {% endif %}
             </ul>
           </li>
         {% endfor %}
       {% endif %}
+
       {% if section.cta and section.cta_link %}
-      <!-- Check for CTA link -->
         <li>
           <a class="p-navigation__dropdown-item p-heading--2"
              style="margin-bottom:0"
              href="{{ section.cta_link }}"
              data-level="1">{{ section.cta }}</a>
         </li>
       {% endif %}
+
       {% if section.main_lists %}
-      <!-- Loop through each of the main lists -->
-        {% for main_list in section.main_lists %}
+        {% for list in section.main_lists %}
           <li class="p-navigation--list-heading">
-            <h2 class="p-muted-heading" data-level="1">{{ main_list.title }}</h2>
+            <h2 class="p-muted-heading" data-level="1">{{ list.link_list.title }}</h2>
```

Comment:
After setting the variable
```suggestion
            <h2 class="p-muted-heading" data-level="1">{{ linked_list.title }}</h2>
```